modules/cli/cmd/devrunner/config/base.go

```diff
@@ -0,0 +1,145 @@
+package config
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/pelletier/go-toml/v2"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/viper"
+)
+
+// BaseConfig is a map of string key-value pairs that represent the base
+// configuration for all services in the Astria stack. The key-values pairs are
+// also parsed into environment variables for the services to consume.
```

Comment:
I think we should mention why we chose to use a `map` here, which was to support dynamically adding configuration values.